lib/charms/layer/nodejs.py

```diff
@@ -1,8 +1,12 @@
 import os
 import sys
+
 from shell import shell
+from charms import layer
```

Comment:
nm i saw the location of the file being moved, so that's the preferred way now is to have everything in ./charms/layer